README.md

```diff
@@ -87,6 +87,7 @@ When using [Go templates](https://golang.org/pkg/text/template/) as part of the
 - `env KEY`: function that returns the KEY from environment.
 - `sum A B`: function that returns the sum of numbers A and B (only for integers).
 - `file PATH`: function that returns the contents of the file at PATH.
+- `glob PATTERN`: function that returns the names of all files matching PATTERN.
```

Comment:
```suggestion
- `glob PATTERN`: function that returns the names of all files matching [glob](https://pkg.go.dev/path/filepath#Match) PATTERN.
```

Reply:
Updated in [e832836](https://github.com/elastic/stream/pull/165/commits/e832836ddc800c414001cd82782b1aa9db3d1252)